supervisord.conf

```diff
@@ -0,0 +1,72 @@
+[supervisord]
+nodaemon=true
+user=root
+
+[program: event_service]
+command=python event_service/main.py
```

Comment:
**suggestion:** you should run sanic using `sanic event_service.main:create_app` if you go with the comment above about not using a dynamic application